specificatie/openapi.yaml

```diff
@@ -313,38 +313,38 @@ paths:
       tags:
         - Adres
       summary: "levert BAG detals van een openbare ruimte"
-      description: "Raadpleeg een actuele openbare ruimte met de identificatie. Een openbare ruimte is een buitenruimte met een naam die binnen één woonplaats ligt, bijvoorbeeld een straat of een plein. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/fields.feature)."
+      description: "Raadpleeg een actuele openbare ruimte met de identificatie. Een openbare ruimte is een buitenruimte met een naam die binnen één woonplaats ligt, bijvoorbeeld een straat of een plein. Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.3.0/features/fields.feature)."
       parameters:
         - $ref: '#/components/parameters/openbareruimteidentificatie'
```

Comment:
In het endpoint '/adressen' is er voor gekozen om parameters zoals 'pandIdentificatie' in lowerCamelCase te definiëren waarom is daar hier bij 'openbareruimteidentificatie' niet voor gekozen?